react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+export default function ToggleSwitch() {
+  const [isClicked, setIsClicked] = useState(false);
+  function handleClick() {
+    setIsClicked(!isClicked);
+  }
+
+  const text = isClicked ? "ON" : "OFF"
+
+  return (
+    <>
+      <div className="background" style={{ backgroundColor: isClicked ? "#6aa84f" : "#e2e2e2" }}>
+        <button className="circle" onClick={handleClick} style={isClicked ? { right: 0 } : { left: 0 }}></button>
```

Comment:
Where's your CSS? I see some in App.css, but don't you need more than that?
Typically, you would put it in the file ToggleSwitch.css and import it here.